src/main/msp/msp_protocol_v2_betaflight.h

```diff
@@ -24,3 +24,5 @@
 #define MSP2_SEND_DSHOT_COMMAND             0x3003
 #define MSP2_GET_VTX_DEVICE_STATUS          0x3004
 #define MSP2_GET_OSD_WARNINGS               0x3005  // returns active OSD warning message text
+#define MSP2_SET_DISPLAY_NAME               0x3006  // sets the display_name pilot config var
+#define MSP2_GET_DISPLAY_NAME               0x3007  // gets the display_name pilot config var
```

Comment:
perhaps this could potentially be a generic "SET TEXT VALUE" method, and then passing the type value, and the length in the structure.

i.e. perhaps 2 bytes, 1 bytes for the value code, i.e. DISPLAY_NAME, and 1 bytes for the length (assuming 255, and 255 is ample)?

Allows for code simplification (though similar could be done by creating a function.

```   
    case MSP2_SET_TEXT:
        {
            // type byte, then length byte followed by the actual characters
            const uint8_t type = sbufReadU8(src);
            const uint8_t arraySize = MIN(MAX_NAME_LENGTH, sbufReadU8(src));

            char* text;
            switch(type) {
                case TEXT_PILOT_DISPLAY_NAME:
                    text = pilotConfigMutable()->displayName;
                    break;
            }

            // test for validity.
            
            memset(text, 0, arraySize);
            for (unsigned int i = 0; i < arraySize; i++) {
                text[i] = sbufReadU8(src);
            }
```

Reply:
Thank you for the great suggestion @blckmn ! :bow: I have implemented it for both the setter (`MSP2_SET_TEXT`) as well as the getter (`MSP2_GET_TEXT`). 

The getter had to be moved into `mspFcProcessOutCommandWithArg` (as it also takes a `type byte` input) - I hope this doesn't go against the idea behind the method itself. In either case - any input is highly appreciated!

P.S. Fun side-note - I found (and hopefully fixed :crossed_fingers:) a bug with the initial implementation related to `memset(text, 0, arraySize);` (where `arraySize` is the length of the new value coming over MSP). This would prevent us from properly setting a _shorter_ value for the property compared to the current one. The `memset` would unset the first few characters (up to the length of the new value) but it would leave the remaining characters as-is (from the previous value) - resulting in a somewhat interesting behavior in the configurator :grin: 
(This was broken from my first iteration but I must have missed it out while testing)